app/views/transactions/index.html.erb

```diff
@@ -0,0 +1,43 @@
+<div style="
+min-height: 60px;
+display: flex;
+align-items: center;
+justify-content: space-between;
+padding: 0 20px;
+border-bottom: 1px solid;
+">
+<strong>
+<%= @category.name%>
+</strong>
+<strong>
+<% total_amount = Transaction.where(category_id: @category.id).sum(:amouunt)%>
+<%= number_to_currency(total_amount, :unit => "$")%>
+</strong>
+</div>
+<div style="text-align: center; padding: 8px 0">
+<h1>Transactions</h1>
+</div>
```

Comment:
- [**Optional**] What do you think of displaying a message if no transactions have been added? I think this'll increase user experience.

Reply:
> [Optional] As requested by the previous reviewer 🤔  What do you think of displaying a message if no transactions have been added? I think this'll increase user experience. This will improve your project complexity 👍🏻